### DISCUSSION TITLE
Firefox Color

### DISCUSSION THREAD

**Discussion by Kaedriz** [soulhotel/FF-ULTIMA#28] [General]
I'm documenting Firefox Color properties, and I have one last property not found.
Do you happen to know what property "Toolbar Field Separator" from Firefox Color does change?
I somehow found another, maybe relate variable: `--toolbarseparator-color`, but from the looks of it, it doesn't change anything.

**Comment by soulhotel** [comment]
In the browser css from `toolbarbutton.css`, its for the book marks tool bar. And nothing else apparently. 

![2024-04-18_14-41](https://github.com/soulhotel/FF-ULTIMA/assets/155501797/d62ba794-74f4-4c0f-8ef7-9ff23b914332)

**Reply by Kaedriz** [reply]
Oh, that would explain it, as I never have bookmark tab open.

**Comment by soulhotel** [comment]
same, gonna be putting 1.7.2 out pretty soon, biggest changes would be

localized linux titlebar buttons.
..
all-global-positioning condensed into one file instead of three
https://github.com/soulhotel/FF-ULTIMA/blob/main.update.branch/theme/all-global-positioning.css
then some progress on the spreadsheets
https://github.com/soulhotel/FF-ULTIMA/blob/main.update.branch/doc/spreadsheet-all-ultima-variables.md

**Reply by Kaedriz** [reply]
Small note for displaying lines of code, you can copy them as hyperlink:
Ex: https://github.com/soulhotel/FF-ULTIMA/blob/main.update.branch/theme/all-global-positioning.css#L9-L28
(change branch to main for release)
But it would still fall short, when file changes. It also could be static, pointing to commit instead, but then you would potentially see an outdated file, if it changed, so there are drawbacks.
